packages/components/src/custom-select-control-v2/test/index.tsx

```diff
@@ -0,0 +1,176 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+/**
+ * Internal dependencies
+ */
+import { CustomSelect, CustomSelectItem } from '..';
+
+describe( 'CustomSelect', () => {
+	it( 'Should be able to select multiple items when provided an array', async () => {
+		const user = userEvent.setup();
+
+		const defaultValues = [
+			'incandescent glow',
+			'ultraviolet morning light',
+		];
+
+		render(
+			<CustomSelect defaultValue={ defaultValues } label="Multi-select">
+				{ [
+					'aurora borealis green',
+					'flamingo pink sunrise',
+					'incandescent glow',
+					'rose blush',
+					'ultraviolet morning light',
+				].map( ( item ) => (
+					<CustomSelectItem key={ item } value={ item }>
+						{ item }
+					</CustomSelectItem>
+				) ) }
+			</CustomSelect>
+		);
+
+		const currentSelectedItem = screen.getByRole( 'combobox', {
+			expanded: false,
+		} );
+
+		expect( currentSelectedItem ).toHaveTextContent(
+			`${ defaultValues.length } items selected`
+		);
+
+		await user.click( currentSelectedItem );
+
+		expect( screen.getByRole( 'listbox' ) ).toHaveAttribute(
+			'aria-multiselectable'
+		);
+
+		defaultValues.map( ( value ) =>
```

Comment:
Nit: Better expression of intent

```suggestion
		defaultValues.forEach( ( value ) =>
```